modules/loader/src/main/scala/com/snowplowanalytics/snowplow/postgres/config/ConfigPreCheck.scala

```diff
@@ -0,0 +1,30 @@
+package com.snowplowanalytics.snowplow.postgres.config
+
+import cats.effect.Sync
+import cats.data.EitherT
+import cats.implicits._
+
+import java.nio.file.{Path, Files}
+import com.snowplowanalytics.snowplow.postgres.config.LoaderConfig.Source.LocalFs
+
+import scala.util.Try
+
+/**
+  * Check if given configuration meets necessary requirements
+  */
+object ConfigPreCheck {
```

Comment:
What would be the behavior if we didn't have this check?

Reply:
If non existing path is given, it would just run without giving any error.